client/src/components/directory/DirectoryPage.jsx

```diff
@@ -46,16 +53,21 @@ const EMPLOYEE_TABLE_HEADER = [
   { key: 'email', title: 'Email' },
 ];
 
-const ALLOWED_USER_ACTIONS = [
-  'add',
-  'mail',
-  'edit',
-  'delete',
-];
-
 const tabs = [
-  { key: 'candidate', title: 'Candidates' },
-  { key: 'interviewer', title: 'Interviewers' },
```

Comment:
Thanks for the reminder 👍

Reply:
Is it to much to ask now that administrators is merged to master. You merge master into this branch and make the change for admin too? This will also help resolve merge conflicts

Reply:
Of course not :D I just finished resolving the merge conflicts. Do you mind taking a look at the changes again?